__tests__/integration.test.js

```diff
@@ -164,3 +164,57 @@ describe("GET /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("POST /api/articles/:article_id/comments", () => {
+  it("POST:201 should respond with posted message when successful", () => {
+    const postJson = {
+      username: "butter_bridge",
+      body: "Hello, this is a comment",
+    };
+
+    return request(app)
+      .post("/api/articles/9/comments")
+      .send(postJson)
+      .expect(201)
+      .then(({ body: { comment } }) => {
+        expect(typeof comment.comment_id).toBe("number");
+        expect(typeof comment.body).toBe("string");
+        expect(typeof comment.article_id).toBe("number");
```

Comment:
Great job - we also know the article id is 9 :)

Reply:
Added additional expect for article id is 9 :)